.github/workflows/flutter-nightly.yml

```diff
@@ -12,4 +12,4 @@ jobs:
     secrets: inherit
     with:
       upload-artifact: true
-      upload-tag: "nightly"
+      upload-tag: "test-wayland-scale-2"
```

Comment:
????

Reply:
Temporary modification.
To avoid nightly confilict.